compiler_opt/rl/constant_value_network.py

```diff
@@ -45,9 +45,8 @@ def __init__(self, input_tensor_spec, constant_output_val=0, name=None):
 
     self._constant_output_val = constant_output_val
 
-  def call(self, observation, step_type=None, network_state=(), training=False):
+  def call(self, inputs, step_type=None, network_state=(), training=False):
```

Comment:
@yundiqian this takes care of 

```
compiler_opt/rl/constant_value_network.py:48:2: W0237: Parameter 'inputs' has been renamed to 'observation' in overridden 'ConstantValueNetwork.call' method (arguments-renamed)
```

Anything I'm missing?

Reply:
the change looks good to me, original warning due to some trivial tf agents hassles